### ISSUE TITLE
The selected index-pattern is not present and No template found for the selected index-pattern

### ISSUE THREAD

**Issue by apandiyan** [wazuh/wazuh-kibana-app#2461]
| Wazuh | Elastic | Rev |
| ----- | ------- | --- |
| 3.13.1   | 7.9.0     | NA |

**Description**
Couldn't be able to access Wazuh app on Kibana due to the error (The selected index-pattern is not present and No template found for the selected index-pattern) while doing health check after upgrading Kibana from 7.6.2 to 7.9.0 with respective Wazuh upgrade from 3.12.3 to 3.13.1.

We are using custom index-pattern (alerts-la-*) and template which is available already since we were using it on 3.12.3.

**Steps to reproduce**

1. Install Wazuh 3.12.3 and Kibana 7.6.2
2. Configure custom index pattern at wazuh.yml file of Wazuh-Kibana app
3. Do upgrade from 3.12.3 to 3.13.1 along-with elastic stack from 7.6.2 to 7.9.0
4. Login to Wazuh-kibana app

**Screenshots**
![Screenshot from 2020-09-12 23-15-38](https://user-images.githubusercontent.com/47891234/93011016-3f1aed80-f5b0-11ea-9ac3-0d0f270f978c.png)
![Screenshot from 2020-09-13 11-00-04](https://user-images.githubusercontent.com/47891234/93011021-4f32cd00-f5b0-11ea-8774-394eb66b6aa5.png)

**Comment by pablotr9** [comment]
Hi @apandiyan! 

Unfortunately this is a known bug that affects 3.13.x versions. This will be fixed in next release, in 3.13.x the index pattern expects to start with the word `wazuh`, so your index pattern should be named `wazuh-{customName}*`. 
Or you can also manually modify the code of the app, restart Kibana and you can have the custom name you want, you just have to edit this file: `/usr/share/kibana/plugins/wazuh/public/react-services/saved-objects.js` 
And edit this line: (https://github.com/wazuh/wazuh-kibana-app/blob/3.13-7.9/public/react-services/saved-objects.js#L25 )
```
        `/api/saved_objects/_find?type=index-pattern&search_fields=title&search=wazuh*`
```
You will have to remove only the `search=wazuh*` 
so the line should look like this:
```
        `/api/saved_objects/_find?type=index-pattern&search_fields=title`
```

Then, remove bundles: `rm -rf  /usr/share/kibana/optimize/bundles` and restart Kibana to recreate them (this can take a few minutes): `systemctl restart kibana`

That's all, please let me know if it helps or if you have any other question!

Best Regards,
Pablo Torres

**Comment by apandiyan** [comment]
This workaround works for me. Thanks
